src/ros/Publisher.lua

```diff
@@ -70,3 +70,19 @@ function Publisher:publish(msg)
     end
     return ret
 end
+
+function Publisher:publish_with_ns(msg, vehicle_ns)
```

Comment:
What about considering the namespace part of the topic name?

Then we wouldn't need a special method I believe?

See also my comment in https://github.com/tud-cor/fs_mod_ros_windows/pull/8#pullrequestreview-752886838

Reply:
Then what about the message which does not need a namespace, e.g. `rosgraph_msgs/Clock`?

Reply:
Then we create the `Publisher` with a topic name without a namespace?

Example:

 - for per-vehicle odom, you create a `Publisher` with the topic name set to `<vehicle_name_with_id>/odom`
 - for clock, create a `Publisher` with the topic name set to `clock`

that would seem to remove the need for a special function right?

Isn't the only difference between *with* and *without* this one extra string?